app/assets/javascripts/subscriptions.coffee

```diff
@@ -100,3 +100,27 @@ $(document).ready ->
         return false if validGranuleQueryParameters.indexOf(parameter) == -1
 
       true
+
+  $('#subscriber').on 'change', ->
+    if $('#subscriber-error').length > 0
+      $('#subscriber').valid()
```

Comment:
Take a look at collection permissions for alternates because it's a select2; search groups/search and order